<launch>
  <arg name="name" default="gelsight"/>
  <arg name="output_path"/>
  <arg name="num_secs" default="30"/>
  <arg name="num_imgs" default="-1"/>

  <group ns="$(arg name)">
    <rosparam command="load" file="$(find gelsight_ros)/config/gelsight.yml"/>
    
    <node name="record" type="record.py" pkg="gelsight_ros" output="screen">
      <param name="output_path" value="$(arg output_path)"/>
      <param name="num_secs" value="$(arg num_secs)"/>
      <param name="num_imgs" value="$(arg num_imgs)"/>
    </node>
  </group>
</launch>
